testing/simapp/utils.go

```diff
@@ -65,7 +65,8 @@ func SimulationOperations(app App, cdc codec.JSONCodec, config simtypes.Config)
 		}
 	}
 
-	simState.Contents = app.SimulationManager().GetProposalContents(simState)
+	//nolint: staticcheck // SA1019: app.SimulationManager().GetProposalContents is deprecated: Use GetProposalMsgs instead. GetProposalContents returns each module's proposal content generator function with their default operation weight and key.
+	simState.LegacyProposalContents = app.SimulationManager().GetProposalContents(simState)
```

Comment:
This is not needed yet because they don't use proposal msgs in simulation. However, when IBC does, if this is not there, it may be confusing why it does not work.

IBC team - Why not using directly the SDK helper btw? https://github.com/cosmos/cosmos-sdk/blob/d3c319418514601a86c68552e8d76d54daa52a03/testutil/sims/simulation_helpers.go#L51-L52